integration_tests/manifest.yaml

```diff
@@ -0,0 +1,15 @@
+product_name: 'lsst_apps'
+build_id: 'b1925'
+refs: ['u/jonathansick/DM-4195']
+requester_github_handle: 'jonathansick'
+doc_repo:
+    url: 'https://github.com/lsst-sqre/pipelines_docs.git'
```

Comment:
We don't have a standard for this but my impression is that 2 space indented yaml is currently more common than 4 space.